mods/d2k/rules/defaults.yaml

```diff
@@ -237,7 +239,7 @@
 	Inherits@1: ^ExistsInWorld
 	AppearsOnRadar:
 		UseLocation: yes
-	HiddenUnderFog:
+	HiddenUnderShroud:
```

Comment:
This makes them visible for *everyone* under the fog, but we want them to be visible to only the owner and their allies.